tests/slr_integration/Orders/__snapshots__/Controller_Test__test_attendee_list_seat_column__single event with 3 regular Ticket attendees__0.snapshot.html

```diff
@@ -76,14 +76,14 @@ <h3>Attendance Overview</h3>
 			<span class="tec-tickets__admin-attendees-attendance-type-heading-total-label">
 				Total			</span>
 			<span class="tec-tickets__admin-attendees-attendance-type-heading-total-amount">
-				3			</span>
+				6			</span>
 		</div>
 	</div>
 	<div class="tec-tickets__admin-attendees-attendance-type-complete">
 		<div class="tec-tickets__admin-attendees-attendance-type-complete-label">
 			Complete:		</div>
 		<div class="tec-tickets__admin-attendees-attendance-type-complete-amount">
-			3		</div>
+			6		</div>
```

Comment:
These seem like they are double of what i would expect... right ?

Reply:
Not something we will deal with now. I confirmed that it's one test bleeding into another. If you run them locally one by one they have the correct numbers.

I will address this in the Tickets Commerce MR.

Reply:
I dealt with it as part of getting this PR merged into master @bordoni